Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -1673,10 +1508,10 @@ - (BOOL)hasExtendedAttributeWithName:(NSString *)attrName {
 }
 
 - (void)addExtendedAttributeWithName:(NSString *)attrName {
-    const char *path = [filePath UTF8String];
+    const char *path = [filePath fileSystemRepresentation];
     const char *name = [attrName UTF8String];
 
-    int result = setxattr(path, name, NULL, 0, 0, 0);
+    int result = setxattr(path, name, "", 1, 0, 0);
```

Comment:
I think we should put a "1" or "true" in there. For any program that simply reads extended attributes, it'll make it easier to understand our attribute.

Reply:
Fixed in 66a88e0.